### DISCUSSION TITLE
Can i preload the model before the image come in?

### DISCUSSION THREAD

**Discussion by M1429027** [PaddlePaddle/PaddleOCR#13464] [Q&A]
As the title,Can i preload the model before the image come in? Loading the modle when everytime i run the code is too slow for me, It takes about 5 sec to load the model and start to ocr.
如標題，我有辦法先將model開起來等待嗎?，每次都要執行程式時才能load model導致我整體的ocr時間過長，我希望可以藉由先開啟模型等待來縮短整體ocr的速度

**Comment by UserWangZz** [comment]
Try to see the workaround in model deployment
